.gitignore

```diff
@@ -63,4 +63,3 @@ crashlytics-build.properties
 __pycache__/
 *.pyc
 out/
-/generateDS/
```

Comment:
I think we still want this directory ignored right? We shouldn't be committing third-party code. Those should be installed via other methods

Reply:
Ahh, I think I was confused. Is this `generateDS` directory for the generated files?